taprpc/mintrpc/mint.proto

```diff
@@ -79,14 +113,20 @@ message MintAssetRequest {
     If true, then the asset will be created with a group key, which allows for
     future asset issuance.
     */
-    bool enable_emission = 2;
+    bool new_grouped_asset = 2;
```

Comment:
Any reason to _not_ move this into `MintAsset`?

Reply:
There was a reason when `MintAsset` was used within `MintingBatch` - now that that isn't the case, these could be moved into `MintAsset`.